src/components/Catalog.vue

```diff
@@ -481,14 +486,14 @@ export default {
                 itemLink.title ||
                 item.id ||
                 itemLink.href,
-              dateAcquired: item.properties.datetime
+              dateAcquired: item.properties.datetime,
+              loading: false
             };
           }
 
           return {
-            to: `/item${this.path}/${this.slugify(itemUrl)}`,
-            title: itemLink.title || itemLink.href,
-            url: itemUrl
+            loading: true,
+            title: 'Loading...',
```

Comment:
Same as the other comment in this file, we are changing a behaviour that might break compatibility with the current operation of the STAC browser, specially with the deletion of `to`